site/styles.css

```diff
@@ -13,6 +13,16 @@
   -webkit-font-smoothing: auto;
 }*/
 
+html {
+  scroll-padding-top: 120px;
+  scroll-behavior: smooth;
+}
+
+span:target {
```

Comment:
Can we remove this anchor styling? It's not meaningful for a _link target_ to be highlighted.

Reply:
My logic here is without it sometimes it doesn't actually jump to the actual anchor exactly — since the offset is not perfect. But at least this way it doesn't _hide_ the anchor with the navbar. I've removed the cool pink styling, though. :((((